### DISCUSSION TITLE
changing world file

### DISCUSSION THREAD

**Discussion by majesto12** [itzg/docker-minecraft-server#3169] [Q&A]
I have a world directory from a minecraft realm, and I'm trying to make a docker server with the world folder from that realm, but when I change the world folder in the data directory and log in, the terrain doesn't load. My inventory items are there but not the actual world. How can I fix this?

**Comment by rediculum** [comment]
use the environment variable "LEVEL_NAME" with the world folder name as value.

**Reply by majesto12** [reply]
I keep getting an AccessDeniedException on server startup and then it exits with code 0

**Reply by majesto12** [reply]
should the value be the world folder location or just the name of the folder and is it LEVEL_NAME or just LEVEL?

**Reply by itzg** [reply]
The server switches to UID 1000 by default, so ensure the files in mounted data directory are owned accordingly or change running user https://docker-minecraft-server.readthedocs.io/en/latest/configuration/misc-options/#running-as-alternate-usergroup-id

Probably just `LEVEL` is what was meant: https://docker-minecraft-server.readthedocs.io/en/latest/configuration/server-properties/#level-world-save-name

**Reply by rediculum** [reply]
> should the value be the world folder location or just the name of the folder and is it LEVEL_NAME or just LEVEL?

Defining `LEVEL `did not work for me, only `LEVEL_NAME`. 
It's just the name of the folder

**Reply by itzg** [reply]
Oh, you're talking about Bedrock edition, which now makes sense coming from a Realms export. (This discussion is posted on the Java edition repo rather than https://github.com/itzg/docker-minecraft-bedrock-server)
